pos_pricelist/static/src/js/models.js

```diff
@@ -207,7 +207,7 @@ odoo.define("pos_pricelist.models", function (require) {
         _Orderline_initialize.apply(this, arguments);
         if (options.product) {
             this.set_unit_price(
-                options.product.price ||
+                options.price ||
```

Comment:
Why not to implement it the same way Odoo does upstream?: https://github.com/odoo/odoo/blob/11.0/addons/point_of_sale/static/src/js/models.js#L1350

Reply:
It's actually what we're doing. Well, different code but same result, isn't it?